pkg/sql/parser/sql.y

```diff
@@ -1230,17 +1231,21 @@ preparable_stmt:
   {
     $$.val = $1.slct()
   }
-| create_stmt
-| drop_stmt
-| alter_table_stmt
 | insert_stmt
 | update_stmt
 | delete_stmt
+
+explainable_stmt:
+  preparable_stmt
+| create_stmt
+| drop_stmt
+| alter_table_stmt
```

Comment:
Please explain why you are excluding create, drop and alter table from the list of preparable statements?